tools/reviewer/web/src/app/dashboard/page-loading/page-loading.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, Input } from '@angular/core';
```

Comment:
I was thinking if page loading doesn't have to do anything with our diffs logic it can be moved to layout folder.

Reply:
I tried, but component from one module can't be used in another.

Reply:
we could put it in the shared module... but then again it's just one component. We can put it off until there are more components like it.